### DISCUSSION TITLE
Significance calculation

### DISCUSSION THREAD

**Discussion by kskovpen** [scikit-hep/cabinetry#480] [Q&A] [answered]
Hello! Is there a method to calculate an observed significance using background-only hypothesis?

**Comment by kskovpen** [comment, marked as answer]
Sorry for spamming - I just managed :)

**Reply by alexander-held** [reply]
Hi @kskovpen, glad to hear you managed! Out of curiosity: were you looking for [`cabinetry.fit.significance`](https://cabinetry.readthedocs.io/en/latest/api.html#cabinetry.fit.significance) which does the calculation from a likelihood ratio (including the mu=0 and mu=floating hypotheses) or some other version?

**Reply by kskovpen** [reply]
Thanks @alexander-held , this is exactly what I did :)
